build/scripts/build_helper

```diff
@@ -32,6 +32,7 @@ OS_DISTRO=$(grep "^ID=" /etc/os-release | sed "s/ID=//" | sed "s/\"//g")
 OS_RELEASE=$(grep "^VERSION_ID=" /etc/os-release | sed "s/VERSION_ID=//" | sed "s/\"//g")
 case "$OS_DISTRO" in
   fedora) OS_BASEDISTRO="fedora"; INSTALLER="dnf"; CMAKE="cmake" ;;
+  rocky)  OS_BASEDISTRO="fedora";  INSTALLER="yum"; CMAKE="cmake3" ;;
```

Comment:
@arora-sagar 

on RHEL8 and I've checked on `rocky-linux` 8.7, `cmake3` == `cmake` 

please remove the 3.

Raphael